doc/dev/developer_guide/basic-workflow.rst

```diff
@@ -90,35 +92,45 @@ For more information on using GitHub, refer to `GitHub Help
 Local environment
 -----------------
 
-In the local environment created in the previous step, you now have a
-copy of the ``master`` branch in ``remotes/origin/master``. Since the fork
+In the local environment created in the previous step, you now have a copy of
+the ``master`` branch in ``remotes/origin/master``. This fork
 (https://github.com/mygithubaccount/ceph.git) is frozen in time and the
 upstream repo (https://github.com/ceph/ceph.git, typically abbreviated to
-``ceph/ceph.git``) is updated frequently by other developers, you will need
-to sync your fork periodically. To do this, first add the upstream repo as
-a "remote" and fetch it
+``ceph/ceph.git``) is updated frequently by other contributors, you must sync
+your fork periodically. Failure to do so may result in your commits and pull
+requests failing to merge because they refer to file contents that have since
+changed.
+
+First, ensure that you have properly configured your local git environment with
+your name and email address.  Skip this step if you have already configured this
+information.
 
 .. prompt:: bash $
 
+   git config user.name "FIRST_NAME LAST_NAME"
+   git config user.email "MY_NAME@example.com"
+
+Now add the upstream repo as a "remote" and fetch it:
+
    git remote add ceph https://github.com/ceph/ceph.git
    git fetch ceph
 
```

Comment:
Let's add the following here:

  .. prompt:: bash $

Reply:
Done.  Usage for command blocks is mostly consistent within a given doc file, but not across them.  Might be nice at some point to harmonize, even if they do end up rendering the same way.